.github/ISSUE_TEMPLATE/react-native-windows--vnext--module-request.md

```diff
@@ -18,5 +18,12 @@ assignees: ''
 - [ ] Issue exists on module repo (if so, [link]())
 <!-- If the community module repo already has an issue tracking adding React Native Support, please check this box and link to it. -->
 
-## Overview of Required Functionality
+## Customer Information
+
+### Which app/customer is requesting this module?
```

Comment:
These seem redundant. And/or Customer Information needs more of a prompt for what sort of information you'd expect (maybe a Contoso example or whatever).

Reply:
I'm using `## Customer Information` as a header, not to request more information.

Reply:
It's probably clearer with fewer headers. Removed in latest commit.